docker-compose.yml

```diff
@@ -45,7 +45,6 @@ services:
       - FLASK_APP=lighthouse
       - FLASK_ENV=development
       - EVE_SETTINGS=development.py
-
```

Comment:
Why remove this line? It adds some separation.

Reply:
I'm just making it consistent with the other services above.  Though I see Crawler also has a blank line here.  Not sure which is preferred but if you prefer to leave it in, I can revert that commit.

Reply:
Oh sorry, I misread where the line was! Yes, this removal is correct to be consistent.